admin/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="payment">
+            <group id="liqpay_section" sortOrder="6" showInDefault="0" showInWebsite="0" showInStore="0">
+                <group id="liqpay" translate="label" type="text" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>LiqPay</label>
+                    <frontend_model>Magento\Paypal\Block\Adminhtml\System\Config\Fieldset\Payment</frontend_model>
+                    <fieldset_css>complex</fieldset_css>
+                    <attribute type="displayIn">other_payment_methods</attribute>
+                    <group id="liqpay_general" translate="label" showInDefault="1" showInWebsite="1" sortOrder="5">
+                        <label>General Settings</label>
+                        <attribute type="expanded">1</attribute>
+                        <frontend_model>Magento\Config\Block\System\Config\Form\Fieldset</frontend_model>
+                        <field id="active" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="0" canRestore="1">
+                            <label>Enabled</label>
+                            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                            <config_path>payment/liqpay/active</config_path>
```

Comment:
Please use payment/pronko_liqpay/active config_path, and check the other paths in this file.